### PR TITLE
expanding CEI-schema for import of Fontenay charters

### PR DESCRIPTION
**Pull request by NTsch**
This makes several additions and changes to the elements in the current schema (e.g. for lacunae, metamarks, etc.), which are used to represent the contents from the Fontenay TEI corpus in CEI. 

Indentations should be better this time, but there's a formatting difference in line length. I can try to adjust this if it causes problems.

### CODE REVIEW

File: my/XRX/src/mom/app/cei/xsd/cei.xsd [hunk 1]
```diff
@@ -1244,13 +1119,12 @@
     <xs:element name="seg">
         <xs:complexType mixed="true">
             <xs:choice minOccurs="0" maxOccurs="unbounded">
+                <xs:element ref="c"/>
                 <xs:element ref="expan"/>
                 <xs:element ref="hi"/>
-                <xs:element ref="damage"/>
```
Comment (StephanMa) [inline my/XRX/src/mom/app/cei/xsd/cei.xsd:1249]: Is there a particular reason why those elements are removed? Is ensured that all documents are valid after merging?

File: my/XRX/src/mom/app/cei/xsd/cei.xsd [hunk 2]
```diff
@@ -4175,9 +4056,13 @@
         </xs:annotation>
         <xs:complexType mixed="true">
             <xs:choice minOccurs="0" maxOccurs="unbounded">
-                <xs:element ref="w"/>
```
Comment (StephanMa) [inline my/XRX/src/mom/app/cei/xsd/cei.xsd:4178]: same here

**Review by StephanMa — COMMENTED** [review]
Please have a look

**Comment by NTsch** [comment]
Thanks for pointing these out, I'll need to double check.

**Comment by NTsch** [comment]
The `w` and `pc` elements I added to `supplied` became unnecessary due to a change in how that element is converted in this corpus, so I removed them.
The other removal was due to elements being part of groups that were already allowed, and therefore became unnecessary, but I changed this to make it clearer in this update.

The formatting is now also the same as in the live version, to make it a bit clearer as well.

**Review by StephanMa — APPROVED** [review]

**Comment by StephanMa** [comment]
lgtm

**Pull request merged**